src/components/NavBar/NavBar.component.jsx

```diff
@@ -73,13 +73,16 @@ function NavBar() {
                   className="formcheck"
                 />
               </Form>
-              {isAuth ? (
-                <a onClick={logOut} href="#">
-                  LogOut
-                </a>
-              ) : (
-                <a href={'/login'}>Login</a>
-              )}
+              <div>
+                {isAuth ? (
+                  <a onClick={logOut} href="#">
+                    LogOut
+                  </a>
+                ) : (
+                  <a href={'/login'}>Login</a>
+                )}
+                <IconImage src={isAuth.avatarUrl}></IconImage>
```

Comment:
Here you must access to `avatarUrl` prop until you retrieve isAuth as **true** or add an option value in case is undefined 🤨